### ISSUE TITLE
[Version Request]

### ISSUE THREAD

**Issue by juanescar** [mysticdrew/journeymap-teams#6]
Are there plans for the 1.21.1 on NeoForge?

**Comment by mysticdrew** [comment]
Yes, but it will require a rewrite. I wont work on this mod again until journeymap 6.0.0 is out of beta.

**Comment by mysticdrew** [comment]
Yes, sorry for the delay, I needed to stuff added to journeymap before I could work on this. 
I would expect a 1.21.1 and 26.1.2 release very soon.
